command/bridge/deposit/erc1155/deposit_erc1155.go

```diff
@@ -227,13 +213,25 @@ func runCommand(cmd *cobra.Command, _ []string) {
 		return
 	}
 
-	outputter.SetCommandResult(
-		&depositERC1155Result{
-			Sender:    depositorAddr.String(),
-			Receivers: dp.Receivers,
-			Amounts:   dp.Amounts,
-			TokenIDs:  dp.TokenIDs,
-		})
+	res := &depositResult{
+		Sender:    depositorAddr.String(),
+		Receivers: dp.Receivers,
+		Amounts:   dp.Amounts,
+		TokenIDs:  dp.TokenIDs,
+	}
+
+	if dp.ChildChainMintable {
+		exitEventID, err := common.ExtractExitEventID(receipt)
+		if err != nil {
+			outputter.SetError(fmt.Errorf("failed to extract exit event: %w", err))
+
+			return
+		}
+
+		res.ExitEventID = strconv.FormatUint(exitEventID.Uint64(), 10)
```

Comment:
Maybe we can keep this data as *big.Int and then outputter should convert it to string

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1560/commits/2a3d80c845b44ca5393a65c6bc78df229ff1e31d